frontend/src/action-creator.sync.js

```diff
@@ -118,35 +113,36 @@ export const updateWorkspaceDetail = workspaceDetail => ({ type: `${UPDATE}/${WO
 export const WORKSPACE_LOADED = `${WORKSPACE}/Loaded`
 export const setWorkspaceLoaded = () => ({ type: `${SET}/${WORKSPACE_LOADED}` })
 
-const MEMBER = 'Member'
-export const WORKSPACE_MEMBER = `${WORKSPACE}/${MEMBER}`
-export const WORKSPACE_MEMBER_LIST = `${WORKSPACE_MEMBER}/List`
-export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${WORKSPACE_MEMBER_LIST}`, workspaceMemberList })
-// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function postWorkspaceMember()
+const ROLE = 'Role'
+export const USER_ROLE = `${USER}/${ROLE}`
+export const USER_ROLE_LIST = `${USER_ROLE}/List`
+export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${USER_ROLE_LIST}`, workspaceMemberList })
+// INFO - CH - 2023-11-22 - USER_ROLE_ADD is used by api function postWorkspaceMember()
 // but nothing is bound to it
-export const WORKSPACE_MEMBER_ADD = `${WORKSPACE_MEMBER}/${ADD}`
-// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function deleteWorkspaceMember()
+export const USER_ROLE_ADD = `${USER_ROLE}/${ADD}`
```

Comment:
this duplicate ADD_USER_ROLE
the action should be the prefix so it's this one that should be removed

Reply:
and its usage in postWorkspaceMember should be refactored

Reply:
done